src/sonic-yang-models/yang-models/sonic-interface.yang

```diff
@@ -68,6 +68,12 @@ module sonic-interface {
 					}
                                         default "0";
 				}
```

Comment:
Can you add Yang model tests for mpls enable/disable on RIFs? Most of tests are in sonic-yang-models/tests/yang_model_tests/tests

Reply:
@kperumalbfn I looked at the types of tests currently under sonic-yang-models.  It looked like it was mostly parameter validation.  Since this mpls knob has two fixed values, I wasn't sure what more needed to be checked for that.  Do you have something specific in mind?  Can you provide a pointer to an example?

Reply:
I think it will be good to have a test for one type of interface. SAI default value for SAI_ROUTER_INTERFACE_ATTR_ADMIN_MPLS_STATE is disabled, we can use that. There are examples for NAT_ZONE in sonic-yang-models/tests/yang_model_tests/tests/interface.json

Reply:
@kperumalbfn  Type for mpls attribute has been changed from string to enum, which I believe should eliminate the need for valid/invalid attribute value tests.  No explicit default should be defined in yang models due to lack of support for this atttribute in many existing platforms.